clinica/pydra/t1_volume/tissue_segmentation/cli.py

```diff
@@ -0,0 +1,28 @@
+import click
+
+import clinica.pydra.engine_utils as pydra_utils
+import clinica.pydra.t1_volume.tissue_segmentation.pipeline as pydra_t1vol
+from clinica.pipelines import cli_param
+
+pipeline_name = "pydra-t1vol-ts"
+
+
+@click.command(name=pipeline_name, hidden=True)
+@cli_param.argument.bids_directory
+@cli_param.argument.caps_directory
+def cli(
+    bids_directory: str,
+    caps_directory: str,
```

Comment:
We are losing all the options from the current CLI here, right? (i.e. things like `tissue_classes`, `dartel_tissues`, `tissue_probability_maps`...). You could build the parameters dictionary here, and pass it to the workflow such that the provided values will be used. Then, using the `set_default` method like you did makes perfect sense. WDYT?

Reply:
Right. I wanted to start with an implementation in its simplest form without the argument because the idea was to have argument construction and "Clinica integration" left for another decorator. 

That being said I'll add the arguments to the CLI in the meantime as you suggested.